Etch.OrchardCore.SEO.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>0.7.0-rc2</Version>
+    <Version>0.7.1-rc2</Version>
```

Comment:
You'll also need to change the version number in the [`Manifest.cs`](https://github.com/EtchUK/Etch.OrchardCore.SEO/blob/master/Manifest.cs#L7) file.